cmd/access/node_builder/staked_access_node_builder.go

```diff
@@ -316,13 +316,13 @@ func (builder *StakedAccessNodeBuilder) enqueueUnstakedNetworkInit() {
 
 		var heroCacheCollector module.HeroCacheMetrics = metrics.NewNoopCollector()
 		if builder.HeroCacheMetricsEnable {
-			heroCacheCollector = metrics.NetworkReceiveCacheMetricsFactory(builder.MetricsRegisterer)
+			heroCacheCollector = metrics.PublicNetworkReceiveCacheMetricsFactory(builder.MetricsRegisterer)
```

Comment:
Shouldn't this also be applied to `unstaked_access_node_builder`?

Reply:
unstaked_access_node_builder is actualy fine, because it only has a single network. So it just used the default one.